src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -918,19 +930,20 @@ private static Symbol ComputeImplementationForInterfaceMember(Symbol interfaceMe
                 MultiDictionary<Symbol, Symbol>.ValueSet explicitImpl = currType.GetExplicitImplementationForInterfaceMember(interfaceMember);
                 if (explicitImpl.Count == 1)
                 {
+                    implementationInInterfacesMightChangeResult = false;
                     return explicitImpl.Single();
                 }
                 else if (explicitImpl.Count > 1)
                 {
                     diagnostics.Add(ErrorCode.ERR_DuplicateExplicitImpl, implementingType.Locations[0], interfaceMember);
+                    implementationInInterfacesMightChangeResult = false;
                     return null;
                 }
 
                 // WORKAROUND: see comment on method.
-                // https://github.com/dotnet/roslyn/issues/34452: Is this workaround still relevant? Do we need it when we are looking
-                //                                                for implementations in interfaces?
                 if (IsExplicitlyImplementedViaAccessors(interfaceMember, currType, out Symbol currTypeExplicitImpl))
```

Comment:
I don't know what this comment refers to.

Reply:
> I don't know what this comment refers to.

What comment? There is no comment on this line.

---
In reply to: [324305838](https://github.com/dotnet/roslyn/pull/38683#discussion_r324305838) [](ancestors = 324305838)

Reply:
`// WORKAROUND: see comment on method.`

---
In reply to: [324321361](https://github.com/dotnet/roslyn/pull/38683#discussion_r324321361) [](ancestors = 324321361,324305838)

Reply:
> ```// WORKAROUND: see comment on method.```

This is an old comment (exists for years). My guess is that refers to the comment on the IsExplicitlyImplementedViaAccessors method. If you would like it to be adjusted in any way, please provide exact wording and I can change it. Or I can remove it. Whichever you prefer.

---
In reply to: [324326868](https://github.com/dotnet/roslyn/pull/38683#discussion_r324326868) [](ancestors = 324326868,324321361,324305838)

Reply:
Can you please replace the "WORKAROUND" with an explanation of the purpose of the code.  I suspect this is a workaround for metadata that exposes implementations for the accessors bur not the property. If I'm right, please say that in the comment.

---
In reply to: [324328421](https://github.com/dotnet/roslyn/pull/38683#discussion_r324328421) [](ancestors = 324328421,324326868,324321361,324305838)

Reply:
I will remove the comment since I do not believe it make sense to duplicate this information. 

---
In reply to: [324331724](https://github.com/dotnet/roslyn/pull/38683#discussion_r324331724) [](ancestors = 324331724,324328421,324326868,324321361,324305838)